packages/editor/src/components/media-placeholder/index.js

```diff
@@ -137,26 +152,25 @@ export class MediaPlaceholder extends Component {
 			allowedTypes = [],
 			hasUploadPermissions,
 		} = this.props;
-
 		const {
 			isURLInputVisible,
 			src,
 		} = this.state;
 
-		let instructions = labels.instructions || '';
-		let title = labels.title || '';
+		let instructions = labels.instructions;
+		let title = labels.title;
 
 		if ( ! hasUploadPermissions && ! onSelectURL ) {
 			instructions = __( 'To edit this block, you need permission to upload media.' );
 		}
 
-		if ( ! instructions || ! title ) {
+		if ( instructions === undefined || title === undefined ) {
```

Comment:
This is likely to break backwards compatibility, isn't it? Folks that have been relying on the default labels will no longer get them.

An alternative would be to accept `null` for these props so that folks can explicitly opt-out of the default label and instructions. It's much less likely folks are using a falsey value other than `undefined` to get the defaults.

Reply:
> This is likely to break backward compatibility, isn't it? Folks that have been relying on the default labels will no longer get them.

I think folks that relied on the defaults will still get them because in that case instructions or title are undefined.

What this allows is the ability to not render instructions or title by setting a null value or empty string in these fields.
There is minor behavior change previously if title/instructions were null or contained an empty string we applied default value now for this cases we don't. But I don't think someone would set the prop to a null/empty string and expect its value to contain the defaults.